ext/standard/scanf.c

```diff
@@ -618,7 +614,7 @@ PHPAPI int php_sscanf_internal( char *string, char *format,
 	if (numVars) {
 		for (i = varStart;i < argCount;i++){
 			if ( ! Z_ISREF(args[ i ] ) ) {
-				php_error_docref(NULL, E_WARNING, "Parameter %d must be passed by reference", i);
+				zend_value_error("Parameter %d must be passed by reference", i);
```

Comment:
As noted on the previous PR, this should be just ZEND_ASSERT, it can't happen.

Reply:
Oh sure, I completely missed (forgot) this comment.